virt/virtual_machines/virt-accessing-vm-ssh.adoc

```diff
@@ -0,0 +1,84 @@
+:_content-type: ASSEMBLY
+[id="virt-accessing-vm-ssh"]
+= Using SSH to access virtual machines
+include::_attributes/common-attributes.adoc[]
+:context: virt-accessing-vm-ssh
+
+toc::[]
+
+You can configure an SSH connection to access virtual machines (VMs) by using the following methods:
+
+* `virtctl ssh` command
++
+You add a public SSH key to a VM before it boots for the first time. Then, you connect to the VM by using the `virtctl ssh` command.
++
+This method is useful for troubleshooting a VM because it requires little time to set up. The connection is slow, but it works consistently even if you are behind a firewall or if the cluster does not have a load balancer.
++
+[NOTE]
+====
+Only the user who created the SSH key pair can use the `virtctl ssh` command.
+====
+
+// dynamic key injection - CNV-30068
+// OpenSSH and virtctl - CNV-26737
+
+* Kubernetes service
++
+You create a service to expose a VM port, add the service to the VM, and then connect by using the `ssh` command.
++
+This method is more time-consuming to set up but it provides better performance than `virtctl`.
+
+[id="connecting-ssh-virtctl_virt-accessing-vm-ssh"]
+== Using virtctl for SSH connections
+
+You create an SSH key pair and add the public key to a virtual machine (VM) before it boots for the first time. Then, you run the `virtctl ssh` command.
```

Comment:
Again, the before boot limitation only applies to the access credentials API and when using the `configDrive`. `virtctl ssh` works with any configured SSH key pairs.

Reply:
Same as https://github.com/openshift/openshift-docs/pull/61419#discussion_r1265344517